packages/frontend/webpack.config.js

```diff
@@ -26,7 +26,22 @@ module.exports = (env) => ({
   module: {
     rules: [
       {
-        test: /\.jsx$/,
+        test: /\.tsx?$/,
```

Comment:
this config too

Reply:
ok so get rid of it altogether and add fresh later if it comes back?
lol now that I have to write that out it! Obviously I would not want to leave those loose ends just in case I want them later